docs/learn/learn-assets.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: learn-assets
-title: Assets
-sidebar_label: Assets
+title: Polkadot Asset Hub
```

Comment:
Can just be Asset Hub. Depending on whether it is on Polkadot Wiki or Kusama Guide, the readers can deduce that we are discussing about Asset Hub on the respective chains.

```suggestion
title: Asset Hub
```